base/templates/layout.html

```diff
@@ -0,0 +1,35 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="title" content="Reddit Clone: {% block meta_title %}{% endblock %}">
+        <meta name="description" content="{% block meta_description %}{% endblock %}">
+        <meta property="og:url" content="{{ request.get_full_path }}" />
+        <meta property="og:type" content="website" />
+        <meta property="og:title" content="Reddit Clone: {% block meta_og_title %}{% endblock %}" />
+        <meta property="og:description" content="{% block meta_og_description %}{% endblock %}" />
+        <meta property="og:image" content="" />
```

Comment:
These [OpenGraph](https://developers.facebook.com/docs/sharing/opengraph) meta tags are what creates the thumbnail and description cards when you post on social media. e.g. Facebook timeline